docs/install.md

```diff
@@ -348,6 +348,48 @@ Select the following tabs for the installation description of your target.
 
     ![Parallels Desktop DietPi startup](assets/images/Parallels2.png){: width="640" height="273" loading="lazy"}
 
+=== "UTM"
+
+    <font size="+2">Introduction</font>
+
+    Virtual machine images are great for those occasions where you want to set up a DietPi system very quickly and test things. Also it may be used as a Debian based Linux system with a small footprint for development purposes, e.g. with the X11 window system. The small footprint makes it optimally usable on PCs without a huge built in RAM. Also several VMs may be run for different applications.
+
+    One big advantage of such a VM is that it needs only a couple of minutes coming to a running DietPi system.
+
+    One of the options of a virtual machine is [__UTM__](https://mac.getutm.app/).
+
+    ![UTM Desktop DietPi machine](assets/images/UTM1.png){: width="900" height="506" loading="lazy"}
+
+    <font size="+2">Prerequisites</font>
+
+    As a starting point you can use an **Apple Mac with macOS 11 Big Sur or higher running the UTM software** (x86 system, e.g. Mac mini 2011/2012/2014/2018 as well as M1 Apple silicon).  
+    On this Mac a free harddisk space of about  
+
+    - 1.2 GiB for a minimal running system  
+    - 5 - 10 GiB for a typical running system with X11  
+
+    is needed. A recommended size is at least a free space of 10 GiB.
+
+    !!! info "Emulation feature of UTM"
+        The emulation feature is an option of the virtualization technique to run x86 code on a M1 Mac system and vice versa a M1 Mac code on a x86 Mac system.  
+        This might be a good choice e.g. for testing purposes, but typically there will be a performance drawback using the emulation instead of running native x86 code on a x86 system resp. native M1 code on a M1 system.
```

Comment:
```suggestion
        This might be a good choice e.g. for testing purposes, but typically there will be a performance drawback using the emulation instead of running native x86 images on an x86 system resp. native M1 images on an M1 system.
```